### ISSUE TITLE
Emerald pouch messages not showing up correctly because of a slight formatting change in Wynncraft

### ISSUE THREAD

**Issue by lxhom** [Wynntils/Wynntils-Legacy#369]
### Version

1.10.5

### Brief Description

When picking up items to the ingredient pouch, it shows up in the ticker if configured, but it doesn't work with the emerald pouch. This is caused by a slightly different formatting. I made a small addition in https://github.com/Wynntils/Wynntils/blob/development/src/main/java/com/wynntils/modules/utilities/events/ClientEvents.java#L310 to log the subtitles.
```java
    @SubscribeEvent
    public void onTitle(PacketEvent<SPacketTitle> e) {
        if (!OverlayConfig.GameUpdate.RedirectSystemMessages.INSTANCE.redirectPouch) return;

        SPacketTitle packet = e.getPacket();
        if (packet.getType() != SPacketTitle.Type.SUBTITLE) return;
        System.out.printf("(LX DEBUG) %s%n", McIf.getUnformattedText(packet.getMessage()));
        if (!McIf.getUnformattedText(packet.getMessage()).matches("^§a\\+\\d+ §7.+§a to pouch$")) return;

        e.setCanceled(true);
        GameUpdateOverlay.queueMessage(McIf.getFormattedText(packet.getMessage()));
    }
```

This shows up in the game output:

```
(LX DEBUG) §a+1 §7Red Mushroom§a to pouch
[01:51:28] [Netty Epoll Client IO #0/INFO] [GameTicker]: Message Queued: §a+1 §7Red Mushroom§a to pouch§r
(LX DEBUG) §a+1§7 Emerald §ato pouch
```

The difference is that its `...\+\d§7 .+ §ato...` instead of `...\+\d §7.+$a to...` (the spaces are moved), which causes the regex to fail. I made a small change to the regex to ignore the position of the space:

```java
        if (!McIf.getUnformattedText(packet.getMessage()).matches("^§a\\+\\d+( §7|§7 ).+( §a|§a )to pouch$")) return;
```

Running it with that line changed fixes the issue.

(I didnt open a PR because I'm super new to MC modding, and I'm really unfamiliar with Gradle and the testing stuff (I barely understand how to build JARs lol), but i still want to contribute to this awesome mod so I reported it here :D)

### Steps to Reproduce

- Turn on Pouch message redirects
- Have an emerald pouch
- Pick up an emerald

### Expected Behaviour

Emerald pickup messages should be in the ticker when picked up into a porch if enabled

### Actual Behaviour

Emerald pickup messages are always a subtitle

### Installed Mods

Only Wynntils

**Comment by ryanzhoudev** [comment]
Already implemented in #354, just needs to be reviewed.

**Comment by lxhom** [comment]
Oh, i completely overlooked that. Nevermind then lol
